src/api-reference/request/v4.endpoints.schemas.markdown

```diff
@@ -45,465 +28,11 @@ If a new or existing project please use version [3.0](./v3.request.html) of this
-
-##### URI Template
-
-```
-GET /v4/expenses/{expenseUuid}
-```
-#### Payload
-
-None
-
-### HTTP Response
-
-#### Payload
-
-[Expense](#schema-expense) - The Expense having {expenseUuid} as unique identifier
-
-### Example
-
-[Example](./v4.examples.html#get-existing-expense-expense-resource)
-
-## <a name="update-content-existing-expected-expense-resource"></a>Update the content of an existing expected Expense
-
-### HTTP Request
-
-#### Parameters
-
-Name|Parameter Type|Data Type|Description
----|---|---|---
-expenseUuid|path|string|**Required** The unique identifier of the Expense to update
-userId|path|string|The unique identifier of the user performing the expense update. <br> **Required** when connecting with a Company token, if empty a 400 `missingRequiredParam` error code will be displayed
-
-##### URI Template
-
-```
-PUT /v4/expenses/{expenseUuid}
-```
-
-#### Payload
-
-[Expense](#schema-expense)
-
-### HTTP Response
-
-#### Payload
-
-[Expense](#schema-expense) - The Expense having {expenseUuid} as unique identifier after update
-
-### Example
-
-[Example](./v4.examples.html#update-content-existing-expected-expense-resource)
-
-## <a name="delete-expected-expense-request-resource"></a>Delete an expected Expense from the Request
-
-### HTTP Request
-
-#### Parameters
-
-Name|Parameter Type|Data Type|Description
----|---|---|---
-expenseUuid|path|string|**Required** The unique identifier of the Expense to delete
-userId|path|string|The unique identifier of the user performing the deletion of the expense. <br> **Required** when connecting with a Company token, if empty a 400 `missingRequiredParam` error code will be displayed
-
-##### URI Template
-
-```
-DELETE /v4/expenses/{expenseUuid}
-```
-
-#### Payload
-
-None
-
-### HTTP Response
-
-#### Payload
-
-None
-
-### Example
-
-[Example](./v4.examples.html#delete-expected-expense-request-resource)
-
-# <a name="travel-agency-resource"></a>Travel Agency Resources
-
-Manage the configuration for travel agencies integrated with Concur Request.
-
-## <a name="get-travel-agency-description-resource"></a>Get the description of a travel agency office
-
-### HTTP Request
-
-#### Parameters
-
-Name|Parameter Type|Data Type|Description
----|---|---|---
-agencyUuid|path|string|**Required** The unique identifier of the Travel Agency
-
-##### URI Template
-
-```
-GET /v4/travelagencies/{agencyUuid}
-```
-
-#### Payload
-
-None
-
-### HTTP Response
-
-#### Payload
-
-Name|Type|Format|Description
----|---|---|---
-emailAddress|string|-|The travel agency email address
-id|string|-|The travel agency unique identifier
-name|string|-|The travel agency office name
-proposalType|string|-|The travel agency proposal type. Possible value: `CWT`, `CWTF`, `AEBT` or `API`
-
-### Example
-
-[Example](./v4.examples.html#get-travel-agency-description-resource)
-
 ## <a name="schema"></a>Schema
 
 ### <a name="schema-request"></a>Request
```

Comment:
The Format mentioned in the Description should go into the Format column, like the date format in approvalLimitDate. RFCs and ISOs shouldn't be external links, as we don't want to deal with maintaining them and people can google if need be. 
Employee should be referred to as "they" instead of "he"
Add periods to definitions and give grammar a once-over.